test/expensive/ekscluster/furyctl_test.go

```diff
@@ -148,18 +148,19 @@ var (
 	_ = Describe("furyctl & distro v1.25.8 - public minimal", CreateAndDeleteTestScenario("1.25.8", false))
 	_ = Describe("furyctl & distro v1.25.9 - public minimal", CreateAndDeleteTestScenario("1.25.9", false))
 
-	_ = Describe("furyctl & distro v1.27.1 - public minimal - ephemeral", CreateAndDeleteTestScenario("1.25.9", true))
+	_ = Describe("furyctl & distro v1.25.9 - public minimal - ephemeral", CreateAndDeleteTestScenario("1.25.9", true))
 
 	_ = Describe("furyctl & distro v1.26.0 - public minimal", CreateAndDeleteTestScenario("1.26.0", false))
 	_ = Describe("furyctl & distro v1.26.1 - public minimal", CreateAndDeleteTestScenario("1.26.1", false))
 	_ = Describe("furyctl & distro v1.26.2 - public minimal", CreateAndDeleteTestScenario("1.26.2", false))
 	_ = Describe("furyctl & distro v1.26.3 - public minimal", CreateAndDeleteTestScenario("1.26.3", false))
 	_ = Describe("furyctl & distro v1.26.4 - public minimal", CreateAndDeleteTestScenario("1.26.4", false))
 
-	_ = Describe("furyctl & distro v1.27.1 - public minimal - ephemeral", CreateAndDeleteTestScenario("1.26.4", true))
+	_ = Describe("furyctl & distro v1.26.4 - public minimal - ephemeral", CreateAndDeleteTestScenario("1.26.4", true))
```

Comment:
Is this duplicated?

Reply:
nope this one tests also ephemeral folder's feature